mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -523,8 +523,8 @@ as "Intel Haswell" or "Intel Sandy Bridge".
 
 <a name="nested_logging_config"></a>The `logging_config` block supports:
 
-*  `enable_components` - (Required) The GKE components exposing logs. Supported values include:
-`SYSTEM_COMPONENTS` and `WORKLOADS`.
+*  `enable_components` - (Optional) The GKE components exposing logs. Supported values include:
```

Comment:
This seems to still be [required](https://github.com/GoogleCloudPlatform/magic-modules/pull/6772/files?diff=unified&w=1#diff-c8cc57371d31377352c614135ade5ab3273f80b595ee86b222b1271ab6d5b984L717).

Reply:
Thanks, changed it to "required"